db/migrations/20190417075636_create_address_token_balances.sql

```diff
@@ -0,0 +1,41 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE OR REPLACE VIEW accounts.address_token_balances AS (
+    SELECT
+        address_hash,
+        token_contract_address_hash,
+        block_number,
+        COALESCE(SUM(amount), 0)
+    FROM (
+        SELECT
+            xfer.src AS address_hash,
+            xfer.contract AS token_contract_address_hash,
+            xfer.block_number AS block_number,
+            -SUM(xfer.amount) OVER (
+                PARTITION BY xfer.src, xfer.contract, xfer.block_number
+                ORDER BY xfer.block_number ASC
+                ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW
+            ) AS amount
```

Comment:
Looks like the Travis test error is because this is still expected to be named `value` over [here](https://github.com/vulcanize/account_transformers/blob/master/transformers/integration_tests/light_account_transformer.go#L228)